src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -276,15 +276,16 @@ export const commandTileIndexValidate = command({
     let retileNeeded = false;
     for (const val of outputs.values()) {
       if (val.length < 2) continue;
+      // FIXME
```

Comment:
@blacha what was the FIXME relating to? Does this code need further updates?

Reply:
we are assuming the tileNames is only one long, we should check

Reply:
now returning false early if length is not equal to 1.